src/window.rs

```diff
@@ -1042,6 +1042,17 @@ impl Window {
     self.window.set_ime_position(position.into())
   }
 
+  /// Sets the taskbar progress state.
+  ///
+  /// ## Platform-specific
+  ///
+  /// - **iOS / Android / macOS:** Unsupported.
+  #[inline]
+  pub fn set_progress_bar(&self, progress: ProgressBarState) {
```

Comment:
it will be something like this
```rs
#[inline]
pub fn set_progress_bar(&self, progress: ProgressBarState) {
	self.p.set_progress_bar(progress)
}
```

and then in `src/platfomr_impl/linux/event_loop.rs` you will have the implementation use  this field to send the request
https://github.com/tauri-apps/tao/blob/9c8613c3ba3cd64dbaa2d9ddbdbe75bae7035d44/src/platform_impl/linux/event_loop.rs#L57

and then you can receive the request here https://github.com/tauri-apps/tao/blob/9c8613c3ba3cd64dbaa2d9ddbdbe75bae7035d44/src/platform_impl/linux/event_loop.rs#L905 and do the same logic

Reply:
Done